### ISSUE TITLE
Write operation omits known object type

### ISSUE THREAD

**Issue by fl-ma** [AaltoAsia/O-MI#14]
Selecting an object in the web UI's tree and performing a read-operation returns this output
`<omiEnvelope xmlns="http://www.opengroup.org/xsd/omi/1.0/" version="1.0" ttl="0">
  <read msgformat="odf">
    <msg>
      <Objects xmlns="http://www.opengroup.org/xsd/odf/1.0/">
        <Object>
          <id>ParkingService</id>
          <Object>
            <id>ParkingFacilities</id>
            <Object>
              <id>BMW Research and Technology House</id>
            </Object>
          </Object>
        </Object>
      </Objects>
    </msg>
  </read>
</omiEnvelope>`

Object "BMW Research and Technology House" is of type mv=ParkingFacility, this is not returned by read-operation. This leads to problems when using this data for later write-operations.

Tested with version 1.0.4

**Comment by TK009** [comment]
This xml that you have given is only the request. You should press `Send` to get the data, including the type.

**Comment by fl-ma** [comment]
sorry, meant to post a write example.

when I want to update an InfoItem, I select it and click on "write". This returns this:
`<omiEnvelope xmlns="http://www.opengroup.org/xsd/omi/1.0/" version="1.0" ttl="0">
  <write msgformat="odf">
    <msg>
      <Objects xmlns="http://www.opengroup.org/xsd/odf/1.0/">
        <Object>
          <id>ParkingService</id>
          <Object>
            <id>ParkingFacilities</id>
            <Object>
              <id>BMW second facility</id>
              <InfoItem name="numberOfOccupiedParkingSpots">
                <value>VALUE_PLACEHOLDER</value>
              </InfoItem>
            </Object>
          </Object>
        </Object>
      </Objects>
    </msg>
  </write>
</omiEnvelope>`

As a user I would expect that I only have to change VALUE_PLACEHOLDER, but sending it then results in this:
`<omiEnvelope ttl="10" version="1.0" xmlns="http://www.opengroup.org/xsd/omi/1.0/">
  <response>
    <result>
      <return description="Invalid request: Only Object with type mv:ParkingFacility are allowed as childs of Objects/ParkingService/ParkingFacilities." returnCode="400">
      </return>
    </result>
  </response>
</omiEnvelope>`

So my expectation would be, that the generated coding is syntactically correct.
Not a major thing though.

**Comment by TK009** [comment]
Yes right, the error comes from the checks of the parking service agent, which are more strict than O-DF standard, but it might be useful feature to add the type automatically if it exists already.
